tests/examples/test_examples.py

```diff
@@ -53,13 +53,20 @@ def run_test(filepath):
     "step,file",
     [
         ("finetuning", "image_classification.py"),
+        # ("finetuning", "object_detection.py"),  # TODO: takes too long.
```

Comment:
would GPU help?

Reply:
Yes, I assume it would help. I cannot even run `fintuning/translation.py` locally (CPU, 16GB) due to the OOM error.